operation/fhir-operation-bulkdata/src/main/java/com/ibm/fhir/operation/bulkdata/util/BulkDataExportUtil.java

```diff
@@ -199,7 +201,14 @@ public Instant checkAndExtractSince(Parameters parameters) {
         return null;
     }
 
-    public List<String> checkAndValidateTypes(Parameters parameters) throws FHIROperationException {
+    /**
+     * processes both the Parameters object and the query parameters
+     * @param parameters
+     * @param queryParameters
+     * @return
+     * @throws FHIROperationException
+     */
+    public List<String> checkAndValidateTypes(Parameters parameters, MultivaluedMap<String, String> queryParameters) throws FHIROperationException {
```

Comment:
In the case of a GET with queryParameters, I think our operation framework automatically converts that into a Parameters object.
I assume we'd feed both of those into this method?  Assuming thats right, does the redundancy cause any issue?

Reply:
Right, it's redundant for GET, I can add a check - with a post - it does not

Reply:
ok, interesting.  maybe
1. add a test for GET to confirm it behaves how we want
2. i open a new issue to discuss/decide whether the framework should combine query parameters with the Parameters in the body
?

Reply:
Sure. sounds fine. I'll add a test

Reply:
Test is added